content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
```

Comment:
I recommend using [glossary shortcodes](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary) for “API server” and possibly also “control plane”.

Reply:
I do not see entries for those in the glossary; are you suggesting I add them?
(There _is_ an entry for "kube apiserver" but there are other apiservers and this feature applies to those too.)

Reply:
That highlights a gap in the docs, so I logged issue #15957.